### ISSUE TITLE
Install?

### ISSUE THREAD

**Issue by nkkollaw** [Place1/react-native-gtk#27]
Is there any way to install a program?

I'm confused as to how similar to a traditionally-developed program this is. When I run the program node-gir actually talks to GTK, right..?

Can someone do an ELI5?

**Comment by ewnd9** [comment]
Do you mean packaging / distribution for end users or simply installing and running this project following README?

**Comment by nkkollaw** [comment]
The former. :-)

I saw that it's in the roadmap but I was just wondering what the current status is. Is what I'm looking at when I run the example app a simulation of GTK, or actual GTK components which the app communicates via node-gir..?

**Comment by clayrisser** [comment]
We might end up building a reusable inno config for Windows and Debian deb setup for Linux

**Comment by ewnd9** [comment]
There are also existing solutions for packaging node projects to deb 

- https://github.com/heartsucker/node-deb
- https://github.com/LeoIannacone/npm2deb
- https://github.com/arikon/npm2debian

And of course https://github.com/zeit/pkg for generating single binary

> Is what I'm looking at when I run the example app a simulation of GTK, or actual GTK components which the app communicates via node-gir..?

Yes, it's actual GTK with native bindings via `node-gir`

**Comment by nkkollaw** [comment]
Nice.
